src/ExternalSearch.Providers.GoogleMaps/Models/PlaceIdResponse.cs

```diff
@@ -3,20 +3,75 @@
 
 namespace CluedIn.ExternalSearch.Providers.GoogleMaps.Models
 {
-    public class PlaceIdCode
+    // Root myDeserializedClass = JsonConvert.DeserializeObject<Root>(myJsonResponse);
+    public class Geometry
     {
+        public Location location { get; set; }
```

Comment:
Inconsistent property names in all the next few classes, it should be CapitalCase and the json parser is intelligent enough to handle the deserialization.

Lines 9 to 73.